Source/Neptune.Web/Views/Shared/SiteLayout.cshtml

```diff
@@ -22,7 +22,7 @@ Source code is available upon request via <support@sitkatech.com>.
 @inherits Neptune.Web.Views.Shared.SiteLayout
 
 @{
-    SetLayout("~/Views/Shared/NavAndHeaderLayout.cshtml", ViewDataTyped);
+    @SetLayout("~/Views/Shared/NavAndHeaderLayout.cshtml", ViewDataTyped)
```

Comment:
I'm not sure I understand what the difference between these is?

Reply:
Me either but I think the inspector prefers this one.